### ISSUE TITLE
quoting issue in github notifications

### ISSUE THREAD

**Issue by HarHarLinks** [Half-Shot/matrix-hookshot#177]
![image](https://user-images.githubusercontent.com/2803622/149985165-2e4c9087-4f37-473c-a415-a3f4a916b89f.png)

for some reason, this is quoted doubly. original comment:

https://github.com/vector-im/element-web/issues/20578#issuecomment-1015574801

event source excerpt: `<blockquote>\n<blockquote>\n<p>I'm finding a difficulty in reproducing this issue, kindly helpsweat_smile</p>\n</blockquote>\n</blockquote>`

**Comment by Half-Shot** [comment]
Yeah, it's a known problem that the quote doesn't sufficiently wrap.
